example-locale_api/README.md

```diff
@@ -0,0 +1,155 @@
+# Appwrite + VueJS =❤️
+
+This example is to showcase [Appwrite's Locale API](https://appwrite.io/docs/client/locale) with [Vue](https://vuejs.org/) by creating a simple web-app that displays your IP, Country,Continent,CountryCode,Continent and Currency.
+
+## Getting Started
+
+To get started quickly we will use vue cli to create the boilerplate that our code will be built on.
+
+### Installation
+
+```shell
+npm i -g @vue/cli
+vue create appwrite-locale-vue
+cd appwrite-locale-vue
+```
+
+While we are in the CLI we will also install the Appwrite Web SDK by running:
+
+```shell
+npm install appwrite
+```
+
+and finally we will launch the React development server with:
+
+```shell
+npm run serve -- --port [port_number_of_api_endpoint]
+```
+
+## Introducing the Appwrite SDK
+
+We'll be working with a single component, that will be the app component itself. But first create a `utils.js` file in which we'll be intializing the sdk. Create `utils.js` in the `src` folder and paste the following code.
```

Comment:
These sentences need to be looked over again.

Reply:
Made some changes, can you review it ?